test/Tests/src/Data/Numbers_Spec.enso

```diff
@@ -585,7 +585,7 @@ spec =
             231.2 . round . should_be_a Integer
             231.2 . round -1 . should_be_a Integer
 
-        Test.specify "Edge cases" <|
+        Test.specify "Edge cases" pending="Re-enable this if the 15-digit restriction is removed" <|
```

Comment:
Do we plan to ever remove this restriction?

Reply:
Nope

Reply:
Then I think we should just remove these tests.